tools/xmlschema.rb

```diff
@@ -72,13 +73,22 @@ def printElem(_file, _spaces, _elem)
       printDocumentation(_file, _spaces+2, _elem.elements["description"].text)
     end
 
-    _file.printf("%*s<xsd:complexType>\n", _spaces+2, "")
+    if _elem.attributes['name'] == "pose"
```

Comment:
We shouldn't encounter `pose` here because it has `type='pose'`

Reply:
removed 75b33b6d6447182b5945832d8296cf883f55c908